index.html

```diff
@@ -14,22 +14,22 @@
         <div class="bpm-display">
           <span class="tempo">120</span>
           <span class="bpm">BPM</span>
+          <div class="tempo-text">Very slow</div>
         </div>
-        <div class="tempo-text">Very slow</div>
         <div class="tempo-settings">
-          <div class="decrease-tempo">-</div>
+          <button class="decrease-tempo">-</button>
```

Comment:
**Comment:** I made these buttons because they're interactive. We can style them however you want just as easily as a `<div>`. We can access them the same in JS, too, `document.querySelector('...')`.

The benefit is that we get basic interaction and accessibility features for free: keyboard interaction, TTS interaction cues, etc., no matter what they look like.

Reply:
makes sense!